plugins/newsdownloader.koplugin/main.lua

```diff
@@ -20,6 +20,35 @@ local FILE_EXTENSION = ".html"
 local FEED_SOURCE_SUFFIX = "_rss_tmp.xml"
 local NEWS_DL_DIR, FEED_CONFIG_PATH
 
```

Comment:
updated, i will make this a user configurable option later so they can change it through a setting.